.buildkite/shared.sh

```diff
@@ -28,8 +28,11 @@ retry() {
 }
 
 is_pr() {
-  [[ "${BUILDKITE_PULL_REQUEST-}" ]] && return
-  false
+  if [ "${BUILDKITE_PULL_REQUEST-}" ] || [ "${BUILDKITE_PULL_REQUEST}" = "false"]; then
+    return true
+  else
+    return false
```

Comment:
I think you've got these backwards?

Reply:
Yup, that actually helped me test it on the PR, just in case